floodviz/static/js/hydrograph.js

```diff
@@ -128,11 +130,20 @@
 					.attr("d", function (d) {
 						return d ? "M" + d.join("L") + "Z" : null;
 					})
-					.on("mouseover", self.mouseover)
-					.on("mouseout", self.mouseout)
-					.on("click", function(d) {return self.click(d.data.key, graph_data)})
+					.on("mouseover", function(d) {
+						options.show_map_tooltip(d.data.name, d.data.key);
+						return self.series_tooltip_show(d);
+					})
+					.on("mouseout", function(d) {
+						options.remove_map_tooltip;
```

Comment:
PyCharm flags this as neither a statement nor a call, are you sure it's correct?